tests/conftest.py

```diff
@@ -0,0 +1,43 @@
+"""Common pytest fixtures."""
+
+from __future__ import (division, absolute_import, print_function,
```

Comment:
Hmm. I dunno. I do this for all my Python 2.7+3 cross-compatible projects just so that I don't get surprises later.